.github/workflows/kata-deploy-push.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: kata-artifacts
-          path: kata-build/kata-static-${{ matrix.asset }}.tar.xz
+          path: kata-build/kata-static-${{ matrix.asset }}.tar
```

Comment:
@jcvenegas Do you know how much time the compression adds? Nonetheless, I agree, there is no need to compress the intermediate steps. Wouldn't the merge script also need to take this into account?

Reply:
I am talking about this : https://github.com/kata-containers/kata-containers/blob/main/tools/packaging/kata-deploy/local-build/kata-deploy-merge-builds.sh#L20
Would that need to be changed as well to drop the .xz?